ams/api/src/test/java/com/netease/arctic/ams/api/metrics/TestTaggedMetrics.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *  *
```

Comment:
There is an extra * in the license, other file is same problem.

Reply:
1

Reply:
The current file and `TestMetricsManager.java` have not been fixed yet.